From 71ebbc4874374b9ffed6fbb7833b211773f796a6 Mon Sep 17 00:00:00 2001
From: Ray Strode <rstrode@redhat.com>
Date: Fri, 20 Feb 2015 10:23:14 -0500
Subject: [PATCH 120/315] gdm-x-session: fix debug message

It says "could not start X server" when failing to start
the dbus-daemon.

https://bugzilla.gnome.org/show_bug.cgi?id=744787
---
 daemon/gdm-x-session.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/daemon/gdm-x-session.c b/daemon/gdm-x-session.c
index b6bfe832..ee02b644 100644
--- a/daemon/gdm-x-session.c
+++ b/daemon/gdm-x-session.c
@@ -407,7 +407,7 @@ spawn_bus (State        *state,
         g_ptr_array_free (arguments, TRUE);
 
         if (subprocess == NULL) {
-                g_debug ("could not start X server: %s", error->message);
+                g_debug ("could not start dbus-daemon: %s", error->message);
                 goto out;
         }
 
-- 
2.15.1

